### DISCUSSION TITLE
New install on Ubuntu wont start fully

### DISCUSSION THREAD

**Discussion by drileyprologic** [akvorado/akvorado#574] [Q&A]
I have ran through the quick start guide and I cant get the akvorado_akvorado-service_1 service to run.  The web interface comes up correctly and I have flows directed at it, but that's it.  Logs from that container

akvorado v1.8.1
  Build date: 2023-03-04T07:46:24+0000
  Built with: go1.19.5
Can be disabled:

- PacketSize
- PacketSizeBucket
- ExporterName
- ExporterGroup
- ExporterRole
- ExporterSite
- ExporterRegion
- ExporterTenant

r

r

- SrcNetMask

- DstNetMask
- SrcNetPrefix
- DstNetPrefix

t

t

- SrcNetName

- DstNetName

- SrcNetRole

- DstNetRole

- SrcNetSite

- DstNetSite
- SrcNetRegion
- DstNetRegion
- SrcNetTenant
- DstNetTenant

- SrcCountry

- DstCountry
- DstASPath
- Dst1stAS
- Dst2ndAS
- Dst3rdAS
- DstCommunities
- DstLargeCommunities
- DstLargeCommunitiesASN
- DstLargeCommunitiesLocalData1
- DstLargeCommunitiesLocalData2
- InIfDescription
- OutIfDescription
- InIfSpeed

- OutIfSpeed
- InIfProvider
- OutIfProvider
- InIfConnectivity
- OutIfConnectivity
- InIfBoundary
- OutIfBoundary
Can be enabled:

n

n
- SrcAddrNAT (main table only)
- DstAddrNAT (main table only)
- SrcPortNAT (main table only)
- DstPortNAT (main table only)
- SrcMAC
- DstMAC

**Comment by vincentbernat** [comment]
That's expected. This service is just here to serve as a template for the other services. That's due to a limitation of older versions of docker-compose.

**Reply by drileyprologic** [reply]
Understood, thanks.  I'm pretty new to docker, but I'm still unable to get flows to show up.  I felt like I worked through the documentation.  Any tips on what to look at?

**Reply by vincentbernat** [reply]
There is a whole "troubleshooting" section.

**Reply by drileyprologic** [reply]
swapped over to sflow and im getting data now.  At least tells me the docker environment is working as expected.  I'll keep messing around with my environment.
